torch/utils/data/dataloader.py

```diff
@@ -30,6 +34,11 @@ def _worker_loop(dataset, index_queue, data_queue, collate_fn):
     global _use_shared_memory
     _use_shared_memory = True
 
+    # Intialize C side signal handlers because Python signal handler on things
+    # like SIGBUS and SIGSEGV will still be executed in C and likely cause error
+    # again.
```

Comment:
I don't think this message is correct. Python signal handlers will cause the error again exactly because the handling will be done on the Python side and not on the C side (the C side only registers that the signal happened and returns)

Reply:
This is a reworded version of Python doc 18.8.1.1

Reply:
But it doesn't match the docs. They say this (emphasis mine):

> **A Python signal handler does not get executed inside the low-level (C) signal handler**. Instead, the low-level signal handler sets a flag which tells the virtual machine to execute the corresponding Python signal handler at a later point

Reply:
Good point. I'll update this comment.